setup.cfg

```diff
@@ -20,12 +20,12 @@ zip_safe = False
 packages=find:
 platforms = any
 install_requires =
-  types-python-dateutil==2.8.19
-  types-PyYAML==6.0.11
-  types-requests==2.28.9
-  python-dateutil==2.8.2
-  requests==2.28.1
-  web3==5.30.0
+  types-python-dateutil>=2.8.19
+  types-PyYAML>=6.0.11
+  types-requests>=2.28.9
+  python-dateutil>=2.8.2
+  requests>=2.28.1
+  web3>=5.30.0
```

Comment:
This is the real change. We should not impose fixed version requirements if this is meant to be used as a dependency (since it limits other projects to depend on it.

Reply:
What happens if there's a big update of one of these packages and it breaks our one?

Reply:
Yes I suppose. Anyway. Let's cross that traps when we get there.

Reply:
I checked requests library for how they do it and they use inequalities 

https://github.com/psf/requests/blob/main/setup.cfg

Reply:
Cool. Good to know that! Thanks for sharing the information.